pkgs/development/tools/kustomize/default.nix

```
@@ -15,14 +15,12 @@ buildGoModule rec {
     owner = "kubernetes-sigs";
     repo = pname;
     rev = "kustomize/v${version}";
-    sha256 = "sha256-7Ode+ONgWJRNSbIpvIjhuT+oVvZgJfByFqS/iSUhcXw=";
+    hash = "sha256-wVdB9fTLYg7Ma0dRgDt7X7ncN0+04DyT8kp2+/aQ018=";
   };
 
   # avoid finding test and development commands
   modRoot = "kustomize";
 
```

Hello

I wanted to try this out and encountered the same issues as already mentioned. Not having a `vendorHash` makes the build fail instantly while adding any `vendorHash` leads to the error mentioned in https://github.com/NixOS/nixpkgs/pull/224075#issuecomment-1500060585.

I found this issue https://github.com/NixOS/nixpkgs/issues/203039 that seems to be the reason for this (kustomize uses go workspaces as well)

I set `proxyVendor = true;`, set the `vendorHash` to `""`, replaced it with the actual hash after trying to build it and ended up with this code that works for me:

```suggestion
  proxyVendor = true;
  vendorHash = "sha256-E7DRRf/izgFQhwSKT3NyG+5gwVOd+y3gzC4Q0aWZbVE=";
```


I just don't know if thats an ideal solution :man_shrugging:

---

I've added your modification and no problems on my side.